### PR TITLE
ddl: correct comments mistake.

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: ddl/ddl_api.go [hunk 1]
```diff
@@ -684,12 +684,13 @@ func (d *ddl) CreateTable(ctx context.Context, ident ast.Ident, colDefs []*ast.C
 	return errors.Trace(err)
 }
 
-// If create table with auto_increment option, we should rebase tableAutoIncID value.
+// handleAutoIncID increases AutoIncID by one. Rebase table's AutoIncID value, when table created with auto_increment option
```
Comment (shenli) [inline ddl/ddl_api.go]: handleAutoIncID handles auto_increment option in DDL. It creates a ID counter for the table and initiates the counter to a proper value. 
For example if the option sets auto_increment to 10. The counter will be set to 9. So the next allocated ID will be 10.

File: ddl/ddl_api.go [hunk 2]
```diff
@@ -704,7 +705,7 @@ func (d *ddl) handleAutoIncID(tbInfo *model.TableInfo, schemaID int64) error {
 	return nil
 }
 
-// Add create table options into TableInfo.
+// handleTableOptions update tableInfo according to table options.
```
Comment (shenli) [inline ddl/ddl_api.go]: update -> updates

File: ddl/ddl.go [hunk 3]
```diff
@@ -150,14 +150,15 @@ func (e *Event) String() string {
 	return ret
 }
 
+// ddl is data definition language which defines basic operation such as create, alter and select.
```
Comment (XuHuaiyu) [inline ddl/ddl.go]: select is not DDL, but DML.

ddl represents the statements which are used to define the database structure or schema.

**Comment by XuHuaiyu** [comment]
@zimulala PTAL

File: ddl/column.go [hunk 4]
```diff
@@ -265,6 +265,7 @@ func (d *ddl) onDropColumn(t *meta.Meta, job *model.Job) error {
 	return errors.Trace(err)
 }
 
+// addTableColumn
```
Comment (coocood) [inline ddl/column.go]: addTableColumn adds a column to the table.

File: ddl/ddl.go [hunk 5]
```diff
@@ -177,7 +178,7 @@ type ddl struct {
 	wait   sync.WaitGroup
 }
 
-// BindStatsHandle will bind the stats handle with a DDL interface. It will be called after a handle has been initialized.
+// RegisterEventCh registers passed channel for ddl Event
```
Comment (zimulala) [inline ddl/ddl.go]: Add `.` at the end of a line.

File: ddl/ddl_worker.go [hunk 6]
```diff
@@ -409,8 +409,8 @@ func toTError(err error) *terror.Error {
 	return terror.ClassDDL.New(terror.CodeUnknown, err.Error())
 }
 
-// For every lease, we will re-update the whole schema, so we will wait 2 * lease time
-// to guarantee that all servers have already updated schema.
+// waitSchemaChanged wait for the completion of updating all servers' schemea. In order to make sure that happens,
+// we wait 2 * lease time.
```
Comment (zimulala) [inline ddl/ddl_worker.go]: s/wait/waits

File: ddl/index.go [hunk 7]
```diff
@@ -466,6 +466,8 @@ type indexTaskOpInfo struct {
 	nextCh    chan int64                 // It notifies to start the next task.
 }
 
+// addTableIndex adds index into table.
+// TODO move this to doc or wiki
```
Comment (zimulala) [inline ddl/index.go]: s/`TODO`/`TODO: `
s/move/Move
Add `.` at the end of a line.

File: ddl/table_test.go [hunk 8]
```diff
@@ -38,7 +38,7 @@ type testTableSuite struct {
 	d *ddl
 }
 
-// create a test table with num int columns and with no index.
+// testTableInfo create a test table with num int columns and with no index.
```
Comment (zhexuany) [inline ddl/table_test.go]: DONE

File: ddl/ddl_api.go (hunk 1 above)
Comment (zhexuany) [inline ddl/ddl_api.go]: DONE

File: ddl/ddl_api.go (hunk 2 above)
Comment (zhexuany) [inline ddl/ddl_api.go]: DONE

File: ddl/ddl.go (hunk 3 above)
Comment (zhexuany) [inline ddl/ddl.go]: DONE

File: ddl/ddl.go (hunk 5 above)
Comment (zhexuany) [inline ddl/ddl.go]: DONE

File: ddl/index.go (hunk 7 above)
Comment (zhexuany) [inline ddl/index.go]: DONE

**Comment by zhexuany** [comment]
@XuHuaiyu @zimulala PTAL

**Comment by zimulala** [comment]
LGTM

File: ddl/ddl_worker.go [hunk 9]
```diff
@@ -409,8 +409,8 @@ func toTError(err error) *terror.Error {
 	return terror.ClassDDL.New(terror.CodeUnknown, err.Error())
 }
 
-// For every lease, we will re-update the whole schema, so we will wait 2 * lease time
-// to guarantee that all servers have already updated schema.
+// waitSchemaChanged waits for the completion of updating all servers' schemea. In order to make sure that happens,
```
Comment (XuHuaiyu) [inline ddl/ddl_worker.go]: s/schemea/schema

**Comment by zhexuany** [comment]
@XuHuaiyu PTAL

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**